config.toml

```diff
@@ -7,6 +7,7 @@ theme = "docdock"
 # themesdir = "themes/"
 pygmentsCodeFences = true
 pygmentsStyle = "monokailight"
+googleAnalytics = "UA-173708573-1"
 
```

Comment:
Is this the secret key?

Reply:
Should we get our own key from Jeremiah?

Reply:
Yes, it is technically something that should be kept private, but after some investigation I couldn't find a way to securely store the value. If you have an suggestions that would be great. For right now I have implemented filters such that only traffic from specific hostnames will be tracked. And yes, whenever you want you get add your own key, but be sure to add a hostname filter.